python/scripts/rfi_receiver.py

```diff
@@ -787,6 +789,73 @@ def http_server2():
     httpd.serve_forever()
 
 
+def rfi_zeroing():
+    # Wait until the correct UTC time (deals with daylight savings time)
+    t_now = datetime.utcnow()
+    t_transit = ephemeris.solar_transit(t_now)[0]
```

Comment:
How is this getting the site coordinates?

Reply:
The `ch_util.ephemeris` routines all know where CHIME is and use it in all calculations. As this is a fairly CHIME specific script I suggested that James just use that.

Reply:
Ah good, thanks!

Reply:
But I think we still need to subtract `downtime/2` minutes correct?

Reply:
As well as fix the other issues you pointed out below.

Reply:
Yeah. Indeed, we need to subtract half of downtime. Good spot as I missed that one!

Reply:
I like this review tag teaming!!!